cloud/pkg/controllermanager/controllermanager.go

```diff
@@ -48,7 +48,7 @@ func NewAppsControllerManager(ctx context.Context) (manager.Manager, error) {
 }
 
 func setupControllers(ctx context.Context, mgr manager.Manager) error {
-	Serializer := json.NewYAMLSerializer(json.DefaultMetaFactory, appsScheme, appsScheme)
+	Serializer := json.NewSerializerWithOptions(json.DefaultMetaFactory, appsScheme, appsScheme, json.SerializerOptions{Yaml: true})
```

Comment:
cc @Congrool @vincentgoat ptal.

Reply:
LGTM, they are equivalent I think.

Reply:
LGTM, please squalish the two commits to one

Reply:
ok, done @fisherxu @Congrool @gy95